extensions/amp-list/0.1/test/test-amp-list-load-more.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpDocService} from '../../../../src/service/ampdoc-impl';
+import {AmpList} from '../amp-list';
+import {Services} from '../../../../src/services';
+import {isExperimentOn, toggleExperiment} from '../../../../src/experiments';
+
+
+describes.realWin('amp-list component', {
+  amp: {
+    ampdoc: 'single',
+    extensions: ['amp-list'],
+  },
+}, env => {
+  let win, doc, ampdoc, sandbox;
```

Comment:
Not sure about the current status of this, but the style guide says only one variable declaration per line.